### PR TITLE
fix #66 warning issued for create_bcgovr_project

### LINKED ISSUE #66: warning when providing custom directory structure
I wanted to setup my own directory structure, specifically for shiny apps, and I get a warning about the length of nzchar(dir_struct)

```r
dirs <-  c("app/", "app/app.R")

bcgovr::create_bcgov_project(path = "C:/testproject",
                             coc_email = "email@gov.bc.ca",
                             dir_struct = dirs)
#> <U+2605> Setting up the testproject project
#> v Setting active project to 'C:/testproject'
#> v Creating 'R/'
#> v Writing a sentinel file '.here'
#> * Build robust paths within your project via `here::here()`
#> * Learn more at <https://here.r-lib.org>
#> v Setting active project to '<no active project>'
#> v Setting active project to 'C:/testproject'
#> v Writing 'README.Rmd'
#> v Writing 'CONTRIBUTING.md'
#> v Writing 'CODE_OF_CONDUCT.md'
#> v Writing 'LICENSE'
#> v Writing '.gitattributes'
#> Warning in if (!nzchar(dir_struct)) dir_struct <- character(0): the condition
#> has length > 1 and only the first element will be used
#> <U+2714> Creating new project
#> <U+2714> Populating with directory structure
#> <U+2714> Adding Apache boilerplate header to the top of 'C:\\testproject/app/app.R'
#> <U+2605> Your new project is created in C:/testproject
```

The issue is that nzchar does not return a single value:

```r
dirs <-  c("app/", "app/app.R")
nzchar(dirs)
#> [1] TRUE TRUE
```

### PR DESCRIPTION
**Pull request by BonnieJRobert**
working on this fix

### CODE REVIEW

**Comment by boshek** [comment]
This looks great. I would add a NEWS item reference yourself and the issue number.

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# bcgovr 1.0.6
```
Comment (boshek) [inline NEWS.md]: For here, I think we might exist in this dev version for a bit. 
```suggestion
# bcgovr 1.0.5.9999
```

**Review by boshek — APPROVED** [review]

File: tests/testthat/test-create_bcgov_project.R [hunk 2]
```diff
@@ -106,6 +106,27 @@ test_that("create_bcgov_project works with empty dir_struct", {
   expect_true(file.exists(file.path(dir, paste0(basename(dir), ".Rproj"))))
 })
 
+test_that("create_bcgov_project works with shiny app dir_struct", {
+  setwd("~")
+  dir <- unique_temp_dir(pattern = "create_foo")
+  ret <- capture.output(create_bcgov_project(dir, coc_email = "me@gov.bc.ca", 
+                                             dir_struct = c("app/", "app.R"),
```
Comment (ateucher) [inline tests/testthat/test-create_bcgov_project.R]: I don't think it matters for this issue but I would have thought the `dir_struct` would be `c("app/", "app/app.R")`

**Review by ateucher — APPROVED** [review]
Looks good to me - a very nice solution! just one small comment on the test `dir_struct`, which doesn't really affect the functionality...

File: tests/testthat/test-create_bcgov_project.R (hunk 2 above)
Comment (BonnieJRobert) [reply tests/testthat/test-create_bcgov_project.R]: Changed this.

**Pull request merged**